test/wrappers/osrm_test.rb

```diff
@@ -162,4 +162,12 @@ def test_large_matrix_split
     assert_equal vector.size, result[:matrix_distance].size
     assert_equal vector.size, result[:matrix_distance][0].size
   end
+
+  def test_ask_for_5k_matrix
```

Comment:
not sure this test could be "unskiped" one day...

Reply:
Ok, I resolved conflicts and moved this test to QA.